openmc/universe.py

```diff
@@ -396,6 +409,22 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
                 height = pixels[0]*px/(params.top - params.bottom)
                 fig.set_size_inches(width, height)
 
+            if outline:
+
+                image_value = [
+                    [get_int_from_rgb(inner_entry) for inner_entry in outer_entry]
+                    for outer_entry in img
+                ]
```

Comment:
I just tried this locally and it didn't work because the `img` data is all floats, whereas `get_int_from_rgb` expects that it is a tuple of ints. I think you need something like:
```Python
rgb = (img * 256).astype(int)
image_value = (rgb[..., 0] << 16) + (rgb[..., 1] << 8) + rgb[..., 2])
```

Reply:
Oh sorry about that, I thought I had checked that but shall get that fix. Thanks for the pointer

Reply:
I understand this error now. I am running without 'libpng' and therefore making ppm plot files. These are loaded up as ints.

![Screenshot from 2023-04-19 11-20-34](https://user-images.githubusercontent.com/8583900/233046175-235a6fd1-6990-4d46-9747-04dac0f4e513.png)

While I assume you have libpng and are loading up png files that appear as floats.

![Screenshot from 2023-04-19 11-21-06](https://user-images.githubusercontent.com/8583900/233046195-767c0af4-e2cf-46d3-ac33-3915d66cca69.png)

I'm sure I can get this working for both cases

Reply:
the last commit should get this working for people with libpng installed and also still work for those without libpng